packages/design-system/src/components/VerticalNav/VerticalNavItemLabel.tsx

```diff
@@ -55,8 +56,8 @@ export const VerticalNavItemLabel = (props: VerticalNavItemLabelProps): React.Re
       'aria-controls': props.subnavId,
       'aria-expanded': !props.collapsed,
       title: props.collapsed
-        ? props.ariaCollapsedStateButtonLabel
-        : props.ariaExpandedStateButtonLabel,
+        ? props.ariaCollapsedStateButtonLabel ?? t('verticalNav.ariaCollapsedStateButtonLabel')
```

Comment:
This ternary got a bit more difficult to follow

Reply:
Refactored the component and changed the translation keys